typedoc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	name: "webpack-cli",
+	mode: "modules",
+	out: "docs",
+	theme: "default",
+	exclude: "*.test.ts",
```

Comment:
we should exclude also .js files? Or the library just parses .ts files?

Reply:
Yeah, we don't have any tests in ts, so this won't affect anything. I'll clean it up anyways.